### PR TITLE
Zavod shuttle destroyed

### PR DESCRIPTION
**Pull request by CactusMouth**
Adds a new Awaysite - **The Zavod Shuttle.**

StrongDMM-2024-04-30 15 49 00

FEATURING ALL NEW:

- Drones and turrets! 
- Decent loot other than just Phoron.
- A cool new place to explore.
- Uses Danger:Reward ratio so you can't just stumble upon an automatic shotgun without a challenge!

Changelog will be added in a few minutes, thanks.

### CODE REVIEW

**Comment by CactusMouth** [comment]
!review

File: maps/away/away_site/zavod_shuttle_destroyed/zavod_shuttle_destroyed.dm [hunk 1]
```diff
@@ -0,0 +1,28 @@
+/datum/map_template/ruin/away_site/zavod_shuttle_destroyed
+	name = "destroyed zavod shuttle"
+	description = "A wrecked shuttle once owned by Zavodskoi Interstellar. It seems to have crashed into a large asteroid."
+
+	prefix = "away_site/zavod_shuttle_destroyed/"
+	suffixes = list("zavod_shuttle_destroyed.dmm")
+
+	sectors = list(ALL_POSSIBLE_SECTORS)
+	spawn_weight = 1
+	spawn_cost = 2
```
Comment (DreamySkrell) [inline maps/away/away_site/zavod_shuttle_destroyed/zavod_shuttle_destroyed.dm:10]: should be = 1

**Comment by DreamySkrell** [comment]
image
wrong exterior space turf
is `/turf/space`
should be `/turf/template_noop`

**Comment by DreamySkrell** [comment]
image
atmos test fail here
this room has air and is exposed to space

**Review by DreamySkrell — CHANGES_REQUESTED** [review]
as above

**Comment by CactusMouth** [comment]
I'll get it fixed in a moment

**Comment by CactusMouth** [comment]
> as above

Fixed.

**Review by DreamySkrell — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**